src/nvim/macros.h

```diff
@@ -183,4 +188,11 @@
 /// @return ((Type *)obj).
 #define STRUCT_CAST(Type, obj) ((Type *)(obj))
 
+// Type of uv_buf_t.len on Windows is ULONG, but others is size_t.
+#if WIN32
+# define UV_BUF_LEN(x)  (ULONG)(x)
+#else
+# define UV_BUF_LEN(x)  (x)
+#endif
```

Comment:
Thanks for address this. libuv is working on it in https://github.com/libuv/libuv/pull/1236 but that will be in version 2...